src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-box-big {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-box-small {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-box-big input {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0%;
+  right: -1.89%;
+  top: 0%;
+  bottom: -15.79%;
+  padding: 0 0 0 62px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  border: 0;
+  margin: 0;
+}
+
+.search-box-small input {
+  align-content: center;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  padding: 0 0 0 33px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 14px;
+  line-height: 19px;
+  border: 0;
+  margin: 0;
+}
+
+.search-btn-big {
+  position: relative;
+  background: transparent;
+  border: 0;
+  padding: 0;
+  top: 24px;
+  left: 25px;
+}
+
+.search-btn-small {
+  position: relative;
+  background: transparent;
+  border: 0;
+  padding: 0;
+  top: 12px;
+  left: 14px;
+}
+
+input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input:focus {
+  background-color: linear-gradient(180deg, #fff 0, #f6f6f7 1);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: bolder;
+  line-height: 22px;
+}
```

Comment:
Do not use `tag `selector. Add class for input and use it

Reply:
Done - ready for re-review.